LICENSE

```diff
@@ -1,6 +1,10 @@
 MIT License
 
-Copyright (c) Rome Tools, Inc. and its affiliates.
+Biome is a fork of Rome Tools (https://github.com/rome/tools).
```

Comment:
I'm not a lawyer, but I think we should play it safe:
- she should keep the **original** license. We can name the file to `ROME_LICENSE`
- generate a new MIT `LICENSE` file, using the template provided (GitHub, or other websites)

Reply:
I think both ways are ok. Anyway, I can make the change.

Reply:
From Wikipedia

> the MIT License also permits reuse within proprietary software, provided that all copies of the software or its substantial portions include a copy of the terms of the MIT License and also a copyright notice.

Let's make a copy, I believe the safest choice